openshift/ess.cm.yaml

```diff
@@ -0,0 +1,51 @@
+---
+apiVersion: template.openshift.io/v1
+kind: Template
+labels:
+  app.kubernetes.io/component: app
+  app.kubernetes.io/instance: "${APP_NAME}-${JOB_NAME}"
+  app.kubernetes.io/managed-by: github
+  app.kubernetes.io/name: nodejs
+  app.kubernetes.io/part-of: "${APP_NAME}-${JOB_NAME}"
+  app: "${APP_NAME}-${JOB_NAME}"
+  template: "chefs-app-event-stream-service-template"
+metadata:
+  name: "chefs-app-event-stream-service"
+objects:
+  - apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: "${APP_NAME}-${JOB_NAME}-event-stream-service"
+    data:
+      FEATURES_EVENTSTREAMSERVICE: ${FEATURE}
```

Comment:
I think the `FEATURES_EVENTSTREAMSERVICE` can also be removed?

Reply:
good catch!